### PR TITLE
feat(core): strip logs from runjob executions and add expand param to executions history

### PR DESCRIPTION
**Pull request by sbwsg**
Run Jobs that generate a large amount of log output can dramatically slow down loading of the pipeline executions screen in deck.

This PR introduces an `expand` query param to the execution history API endpoint.  When `false` the log output under jobStatus in both the context and outputs will be stripped.  The param defaults to true to maintain backwards compatibility.

### CODE REVIEW

**Comment by lwander** [comment]
fyi @anotherchrisberry this is meant to be extended to cut out more details when needed, but for now only addresses a very particular property in the pipeline executions that's grown very large.

File: orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy [hunk 1]
```diff
@@ -330,6 +332,15 @@ class TaskController {
       executionRepository.retrievePipelinesForPipelineConfigId(it, executionCriteria)
     }).subscribeOn(Schedulers.io()).toList().toBlocking().single().sort(startTimeOrId)
 
+    if (!expand) {
+      allPipelines.each { pipeline ->
+        pipeline.getStages().each { stage ->
```
Comment (lwander) [inline orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy]: Should we filter on the stage type? Just to make sure this only hits the run jobs stages?

Comment (sbwsg) [reply orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy]: great idea

Comment (sbwsg) [reply orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy]: done

**Comment by anotherchrisberry** [comment]
👍

**Pull request merged**